resources/public/api/conf/1.0/employment_expenses_retrieve.yaml

```diff
@@ -0,0 +1,108 @@
+summary: Retrieve Employment Expenses
+description: |
+  This endpoint enables you to retrieve existing employment expenses.
+  A National Insurance number and tax year must be provided.
+  
+  ### Test data
+  <p>Scenario simulations using Gov-Test-Scenario headers are only available in the sandbox environment.</p>
+  <table>
+      <thead>
+          <tr>
+              <th>Header Value (Gov-Test-Scenario)</th>
+              <th>Scenario</th>
+          </tr>
+      </thead>
+      <tbody>
+        <tr>
+            <td><p>N/A - DEFAULT<br><strong>?source=user</strong></p></td>
+            <td><p>Requesting <strong>user</strong> source simulates success response with User provided financial details.</p></td>
+        </tr>
+        <tr>
+            <td><p>N/A - DEFAULT<br><strong>?source=hmrcHeld</strong></p></td>
+            <td><p>Requesting <strong>hmrcHeld</strong> source simulates success response with HMRC held financial details.</p></td>
+        </tr>
+        <tr>
+            <td><p>N/A - DEFAULT<br><strong>?source=latest</strong></p></td>
+            <td><p>Requesting <strong>latest</strong> source simulates success response with Latest financial details submitted.</p></td>
+        </tr>
+        <tr>
+            <td><p>NOT_FOUND</p></td>
+            <td><p>Simulates the scenario where no data is found.</p></td>
+         </tr>
+      </tbody>
+  </table>
+
+tags:
+  - Employment Expenses
+
+parameters:
+  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+  - $ref: './common/pathParameters.yaml#/components/parameters/taxYear'
+  - $ref: './common/queryParameters.yaml#/traits/source'
```

Comment:
if my suggestion for updating the `queryParameters.yaml` file is implemented then this will need updating too